### DISCUSSION TITLE
attachmentCardView.addButton and added icons

### DISCUSSION THREAD

**Discussion by hpmayank** [InboxSDK/InboxSDK#1034] [Q&A]
**Question(s)**
Hello,
I attempted to add a button to the attachment by utilizing the [registerThreadViewHandler](https://inboxsdk.github.io/inboxsdk-docs/conversations/#registerthreadviewhandlerhandler) and [attachmentCardView.addButton method](https://inboxsdk.github.io/inboxsdk-docs/conversations/#addbuttonbuttondescriptor). Although the icon was successfully added to the attachment card, I encountered an issue where the newly added button did not align with the buttons added through Gmail's native functionality. Please see the attached screenshot for the misaligned icons.
<img width="210" alt="misaligned-icon" src="https://github.com/InboxSDK/InboxSDK/assets/21031916/d6896365-47e4-41b2-ae4d-fe5feb934fbe">

Do you have any recommendations on how we can resolve this alignment issue and potentially style the button more in line with Gmail's existing buttons?

**Comment by wegry** [comment]
Hey @mmayank28, a minimal repro extension would be helpful in this case. https://github.com/InboxSDK/InboxSDK/blob/main/examples/attachment-card/content.js might be enough to help you start one.

**Reply by hpmayank** [reply]
Hi 
Thanks for the response, I tried to write a minimal extension and code is at repo- https://github.com/mmayank28/inbox-sdk-attachment-card
It would be great if you can take a look

**Reply by hpmayank** [reply]
Hi @wegry 
Is there any updated on above? 
Please inform me if there are any requirements from my side.
